controller/src/inruptController.ts

```diff
@@ -0,0 +1,24 @@
+import { Controller } from "./classes/Controller";
+import { PublicManager } from "./classes/permissionManager/inrupt/PublicManager";
+import { WebIdManager } from "./classes/permissionManager/inrupt/WebIdManager";
+import { InruptStore } from "./classes/stores/InruptStore";
+import { PublicResolver } from "./classes/subjectResolvers/Public";
+import { WebIdResolver } from "./classes/subjectResolvers/WebId";
+import { PublicSubject, WebIdSubject } from "./types/subjects";
+
+export const inruptController = new Controller<{
```

Comment:
minor: i'd just call this the controller (or ActiveController, or ConfiguredController, or whatever). It happens to use only inrupt-dependent functions, but that's 'by accident': if new (non-inrupt) functions get added, we'll change this file